### PR TITLE
schutzbot: enable a subset of tests on Fedora 33 aarch64

### PR DESCRIPTION
**Pull request by ondrejbudai**
This PR adds aarch64 testing to Schutzbot. It's mostly an MVP but hey, it's definitely something that we can iterate on.

The most important part of this PR is the commit that fixes the registration of aarch64 in EC2 - this was sadly broken in the production code.

### CODE REVIEW

**Review by larskarlitski — DISMISSED** [review]
Thanks! Looks correct to me.

**Review by msehnout — CHANGES_REQUESTED** [review]
Looks very nice! CI is unhappy because of unbound variables, but that is easy to fix.

File: schutzbot/Jenkinsfile [hunk 1]
```diff
@@ -240,6 +253,38 @@ pipeline {
...
+                        always {
+                            preserve_logs('fedora33-aarch64-base')
+                        }
+                    }
+                }
+                stage('F33 aarch64 Image') {
+                    agent { label "f33cloudbase && aarch64 && aws" }
+                    environment {
+                        TEST_TYPE = "image"
+                        AWS_IMAGE_TEST_CREDS = credentials('aws-credentials-osbuild-image-test')
```
[the remark this answers is not in the document]
Comment (ondrejbudai) [reply schutzbot/Jenkinsfile:272]: Thanks, fixed!

**Review by teg — APPROVED** [review]

**Review by msehnout — APPROVED** [review]
Very nice!

**Pull request merged**